setup.py

```diff
@@ -2,8 +2,4 @@
 
 from setuptools import setup
 
-setup(
-    setup_requires=["pbr"],
-    pbr=True,
-    version='{{VERSION_PLACEHOLDER}}'
-)
+setup(setup_requires=["pbr"])
```

Comment:
I had to change the invocation here, because if any arguments other than `setup_requires` was specified, PBR attempts to load the non-existing setup.cfg

Reply:
Thanks for pointing this out. I'm not sure if this changes how we were publishing to pypi, but I'll figure that out after merging.